plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -197,9 +179,12 @@ class NewCreateServiceModalForm extends Component {
   }
 
   getAppConfig(currentState = this.state) {
-    let {baseConfig, batch} = currentState;
-    let patch = batch.reduce(jsonConfigReducers, {});
-    return CreateServiceModalFormUtil.applyPatch(baseConfig, patch);
+    let {batch} = currentState;
+    return batch.reduce(this.props.jsonConfigReducers, {});
```

Comment:
Ideally I'd rather not `filter` baseConfig but I most probably should do it for secrets 🤔